lib/cdn_experiment/application_helper_extension.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CdnExperiment
+  module ApplicationHelperExtension
+    extend ActiveSupport::Concern
```

Comment:
Any need for this? Are we actually using any features of 'Concern'?

Reply:
Force of habit :slightly_smiling_face: Since we’re a Rails app, it’s simpler to just use concerns when extending/patching other classes. It provides a nice API (`prepended do`, `included do`, `class_methods do`) and handles properly module dependencies (https://api.rubyonrails.org/v7.1.3.2/classes/ActiveSupport/Concern.html).